### ISSUE TITLE
Show figure captions

### ISSUE THREAD

**Issue by svenboulanger** [marktext/marktext#3464]
### Describe your feature request

In many cases I need to see the alt text for images, especially since that alt text is used as captions by Pandoc. Right now, editing the image has a tiny edit box for alt text which is lacking for any meaningful caption text.

I propose to have the alt and title text of an image visible directly below the image, preferably from there I'd also like to be able to edit it. If this would clutter the visuals too much, perhaps a preference to have it only show when the mouse hovers the image would also work.

### What problem does this feature solve? [optional]

Captions often contain more information in our technical documents, and it also contains the `\label{...}` definitions which I should see if I want to refer to an image.

[edit] May be similar to #2101. I couldn't really figure out what the link with Ulysses and figure numbering was.
